src/MainContainer/MainContainer.js

```diff
@@ -0,0 +1,112 @@
+import React, { Component } from "react";
+import MoviesList from "src/MoviesList";
+import Search from "src/Search";
+import SearchByType from "src/SearchByType";
+import SortByCategory from "src/SortByCategory";
+import Header from "src/Header";
+import Footer from "src/Footer";
+import cx from 'classnames';
+import globalStyles from '../../node_modules/bootstrap/dist/css/bootstrap.min.css';
+import styles from "./MainContainer.css";
+import { bindActionCreators } from "redux";
+import { connect } from "react-redux";
+import {
+  requestApiData,
+  clickStoreData,
+} from "src/redux/actions";
+
+class MainContainer extends Component {
+  constructor(props) {
+    super(props);
+  }
+
+  filterMovies = (searchText, activeSearch, allMovies) => {
```

Comment:
I don't know exactly, but mb it should be filtered by BE API you use? just curious